package.json

```diff
@@ -512,7 +513,8 @@
                     "azureResourceGroups.groupBy": {
                         "type": "string",
                         "description": "%azureResourceGroups.groupBy%",
-                        "default": "resourceType"
+                        "default": "resourceType",
+                        "scope": "machine"
```

Comment:
I changed these scopes to be only machine/user scope. The reason being, you'd get some weird behavior if there was a workspace setting. The command that we have that changes the groupBy setting would set it on the global level however, so it was making it not actually do anything.

I think it makes more sense to have it on the user level since it impacts your Azure Resources view, which really shouldn't be workspace dependent anyway.